app/templates/index.hbs

```diff
@@ -22,7 +22,8 @@
     <a href="https://github.com/ember-engines" rel="noopener" target="_blank" class="content-nav_link">GitHub</a>
   </nav>
 
-  <div class="markdown">
-    {{md-text text=(fr-markdown-file "index") typographer=true}}
-  </div>
+  {{markdown-to-html
+    markdown=(fr-markdown-file "index")
+    extensions="highlight"
```

Comment:
we need of hightlightjs?

Reply:
Please use `broccoli-static-site-json` instead of a markdown resolver

https://github.com/ember-learn/ember-website/blob/master/lib/content-data-generator/index.js#L69

Reply:
@villander see question in discord; I think we should continue conversation there.